enhancements/baremetal/coreos-image-in-release.md

```diff
@@ -0,0 +1,258 @@
+---
+title: coreos-image-in-release
+authors:
+  - "@zaneb"
+reviewers:
+  - "@hardys"
+  - "@dtantsur"
+  - "@elfosardo"
+  - "@sadasu"
+  - "@kirankt"
+  - "@asalkeld"
+  - "@cgwalters"
+  - "@aravindhp"
+  - "@jlebon"
+  - "@dhellmann"
+  - "@sdodson"
+  - "@LorbusChris"
+approvers:
+  - "@hardys"
+  - "@aravindhp"
+creation-date: 2021-09-22
+last-updated: 2021-09-22
+status: implementable
+see-also:
+  - "/enhancements/coreos-bootimages.md"
+---
+
+# Include the CoreOS image in the release for baremetal
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The baremetal platform is switching from a QCOW2 format for its CoreOS image to
+an ISO. To ensure that existing disconnected clusters can update to this, the
+ISO image will be included in the release payload. This will be balanced out by
+removing the RHEL image currently shipped in the release payload. There are
+many other nice benefits, such that after future upgrades clusters will
+provision new machines with the latest CoreOS, without requiring an extra
+reboot.
```

Comment:
We might want to be a bit more specific here, e.g we're switching from using the openstack QCOW2 image to using *the* live-iso that's already supported for UPI baremetal deployments.

And relatedly, another benefit we could call out is that this aligns the baremetal IPI flow with what is already supported for UPI.